op-e2e/setup.go

```diff
@@ -658,6 +661,11 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		return nil, fmt.Errorf("unable to start l2 output submitter: %w", err)
 	}
 
+	batchType := derive.SingularBatchType
```

Comment:
```suggestion
	var batchType uint = derive.SingularBatchType
```

And drop explicit casting below.